### PR TITLE
Connecting to Vehicles AAT update

### PR DESCRIPTION
**Pull request by smcveigh941**

### CODE REVIEW

File: src/main/java/uk/gov/dvsa/mot/fixtures/mot/TesterDoesStepDefinitions.java [hunk 1]
```diff
@@ -396,6 +396,24 @@ public TesterDoesStepDefinitions(WebDriverWrapper driverWrapper) {
...
+        And("^I check the dangerous failures section of the brake test summary of the test summary "
+                        + "does not have \"([^\"]+)\"$",
+                (String text) -> assertFalse(driverWrapper.getTextFromDefinitionListOfIndex("Dangerous failures", 1)
+                        .contains(text)));
+
+        And("^I check the major failures section of the brake test summary of the test summary "
+                        + "does not have \"([^\"]+)\"$",
+                (String text) -> assertFalse(driverWrapper.getTextFromDefinitionListOfIndex("Major failures", 1)
+                        .contains(text)));
+
```
Comment (karlbeckettbjss) [inline src/main/java/uk/gov/dvsa/mot/fixtures/mot/TesterDoesStepDefinitions.java:416]: These new step definitions will need adding to docs/step-definitions.md

Comment (smcveigh941) [reply src/main/java/uk/gov/dvsa/mot/fixtures/mot/TesterDoesStepDefinitions.java:416]: Added these :)

File: src/main/resources/features/theme 1/05a - tester does.feature [hunk 2]
```diff
@@ -55,10 +55,12 @@ Feature: 05a - Tester does...
     And I check the vehicle summary section of the test summary has "Result" of "FAIL"
     And I check the brake test summary section has "Brake results overall" of "Fail"
     And I check the defect section has "Dangerous failures" with value "None recorded"
-    And I check the major failures section of the test summary has "Parking brake efficiency below requirements"
+    And I check the major failures section of the test summary has "None recorded"
     And I check the defect section has "Minors" with value "None recorded"
     And I check the defect section has "PRS" with value "None recorded"
     And I check the defect section has "Advisory text" with value "None recorded"
+    And I check the dangerous failures section of the brake test summary of the test summary has "None recorded"
+    And I check the major failures section of the brake test summary of the test summary has "Service brake efficiency below requirements"
```
Comment (karlbeckettbjss) [inline src/main/resources/features/theme 1/05a - tester does.feature:63]: Is this a new section added to the test summary screen for all users?

Comment (smcveigh941) [reply src/main/resources/features/theme 1/05a - tester does.feature:63]: Yes, brake test defects are now displayed under the brake test section. Before they were under the defects section. See BL-9365

File: src/main/resources/features/theme 1/05f - tester does.feature [hunk 3]
```diff
@@ -55,8 +57,10 @@ Feature: 05f - Tester does...
     And I check the VIN {vin1} is shown within the VIN span text
     And I check the vehicle summary section of the test summary has "Result" of "FAIL"
     And I check the brake test summary section has "Brake results overall" of "Fail"
-    And I check the major failures section of the test summary has "Parking brake efficiency below requirements"
+    And I check the dangerous failures section of the brake test summary of the test summary has "None recorded"
+    And I check the major failures section of the brake test summary of the test summary has "Parking brake efficiency below requirements"
     And I check the dangerous failures section of the test summary has "None recorded"
+    And I check the major failures section of the test summary has "Parking brake efficiency below requirements"
```
Comment (karlbeckettbjss) [inline src/main/resources/features/theme 1/05f - tester does.feature]: Should this be here, it currently fails during testing?

**Pull request merged**